environments/values-usdfdev.yaml

```diff
@@ -0,0 +1,71 @@
+environment: usdfdev
+fqdn: usdf-rsp-dev.slac.stanford.edu
+vault_path_prefix: secret/rubin/usdf-rsp-dev
+# butlerRepositoryIndex: "s3://butler-us-central1-repo-locations/data-repos.yaml"
+repoURL: https://github.com/slaclab/phalanx.git
+
+alert_stream_broker:
+  enabled: false
+cachemachine:
+  enabled: true
+cert_manager:
+  enabled: false
+datalinker:
+  enabled: true
+exposurelog:
+  enabled: false
+gafaelfawr:
+  enabled: true
+hips:
+  enabled: false
+ingress_nginx:
+  enabled: false
```

Comment:
How are these managed?

Reply:
I'm not sure what you're asking, but it's the ingress-nginx application, for which you did add configuration. It needs to be enabled here so that Argo CD will install it and manage it.